templates/taxi/driver_list.html

```diff
@@ -7,7 +8,11 @@ <h1>
         Create
       </a>
     </h1>
-
+    <form action="" method="get" class="form-inline">
+    {{ search_form| crispy  }}
+    <input class="btn btn-secondary" type="submit" value="🔎">
+    </form>
```

Comment:
Create additional template for this code and remove code duplicate instead use include